garage/envs/wrappers/resize_and_gray_scale.py

```diff
@@ -0,0 +1,36 @@
+"""This module implements the resize and grayscale wrapper for gym env."""
+
+import gym
+from gym.spaces import Box
+import numpy as np
+from scipy.misc import imresize
+
+
+class ResizeAndGrayscale(gym.Wrapper):
+    """
+    Resize and grayscale wrapper.
+
+    Gym env wrapper for resizing to (w, h) and converting frames to
+    grayscale.
+    """
+
+    def __init__(self, env, w, h):
+        """Construct the class."""
+        super(ResizeAndGrayscale, self).__init__(env)
+        self.observation_space = Box(0.0, 1.0, shape=[w, h], dtype=np.float32)
+        self.w = w
+        self.h = h
+
+    def _observation(self, obs):
+        obs = imresize(obs, (self.w, self.h))
+        obs = np.dot(obs[:, :, :3], [0.299, 0.587, 0.114]) / 255.0
```

Comment:
Please find a library function for this. If you really need to, define a utility function and write a test for it.

I think scikit-image should have what you need

Reply:
Skimage is not installed in garage. Should it?

Reply:
If you find the function in scikit-image and you decide to use it, you can add scikit-image to environments.yml.

Reply:
scikit uses a different weighting as opencv for RGB = `0.2125 R + 0.7154 G + 0.0721 B`
http://scikit-image.org/docs/dev/api/skimage.color.html
I used scikit since it is a smaller dependency comparing with opencv.

Reply:
![result](https://user-images.githubusercontent.com/10270515/49034831-e3c40500-f167-11e8-8373-adbec73a89ee.png)
There is a slight difference but I guess it is fine.

Reply:
interestingly i was able to see the difference on my phone display but not my laptop. macbooks are calibrated, so perhaps there is no chroma difference for a calibrated display.

anyway, i don't think it should make any difference for learning.